src/Ledger/Utxo/Properties/PoV.lagda

```diff
@@ -0,0 +1,67 @@
+\begin{code}[hide]
+{-# OPTIONS --safe #-}
+
+open import Ledger.Abstract
+open import Ledger.Transaction
+
+module Ledger.Utxo.Properties.PoV
+  (txs : _) (open TransactionStructure txs)
+  (abs : AbstractFunctions txs)
+  where
+\end{code}
+% If the module name changes, change the following macro to match!
+\newcommand{\UtxoPoV}{Utxo/Properties/PoV}
+
+\begin{code}[hide]
+open import Ledger.Certs govStructure
+open import Ledger.Prelude
+open import Ledger.Utxo txs abs
+open import Ledger.Utxo.Properties txs abs using (φ; module DepositHelpers)
+open UTxOState; open Tx; open TxBody
+\end{code}
+
+\begin{lemma}[\LedgerMod{\UtxoPoV.lagda}{\AgdaModule{\UtxoPoV{}}}: \UTXO{} rule preserves value]
+  \begin{itemize}
+    \item \textit{Informally}.
+    Let \ab{s} and \ab{s'} be \UTxOState{}s, let \ab{tx}~:~\Tx{} be
+    a fresh transaction with withdrawals \ab{txwdrls}, and suppose
+    \ab{s}~\AgdaDatatype{⇀⦇}~\ab{tx}~\AgdaDatatype{,UTXO⦈}~\ab{s'}.
+    If \ab{tx} is valid, then the coin value of \ab{s'} is equal to the sum of the
+    coin values of \ab{s} and \ab{txwdrls}.  If \ab{tx} is not valid, then the coin
+    values of \ab{s} and \ab{s'} are equal.  
+    We can express this more formally as follows:
+    \\[4pt]
+    \AgdaField{getCoin}~\ab{s}
+    + \AgdaFunction{φ}(\AgdaField{getCoin} \ab{txwdrls} , \ab{tx}~.\AgdaField{isValid})
+    $≡$ \AgdaField{getCoin} \ab{s'},
+    \\[4pt]
+    where \AgdaFunction{φ}~:~$ℕ$~×~\AgdaDatatype{Bool}~→~$ℕ$ is the function that returns
+    the first argument when the second argument is true, and returns zero when the second
+    argument is false. That is,
+        \[
+          \mbox{\AgdaFunction{φ}} (n , b) = \left\{
+            \begin{array}{ll}
+              n & \text{if } b = \mbox{true}, \\
+              0 & \text{if } b = \mbox{false}.
+            \end{array}\right.
+        \]     
+    \item \textit{Formally}.
```

Comment:
I'd get rid altogether of the phi function; it only adds noise and unnecessary indirection.

Reply:
Do you mean get rid of it in the code as well?  ...using an `if...then...else`?

Reply:
I seem to recall, previously, we had two separate properties, one was proved under the assumption that the transaction is valid, and the other.. invalid.  I think we decided it would be better to be more concise and use a function like phi.  (Perhaps it would have been better to name it chi instead of phi, but I don't think that really addresses your main concern.)

Reply:
I've decided to remove the formal definition of the phi function and rename it chi (since it's sort of a characteristic function).